dripline.html

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Dripline
+menu_title: Dripline
+order: 2
+---
+{% assign posts = site.posts %}
```

Comment:
Missing explainer (See google doc)

A dripline signifies the area under the outermost branches of a tree. It’s this [interface](https://permacultureprinciples.com/principles/_11/) between worlds where interesting and productive things happen.